### ISSUE TITLE
[BUG] New Linux build errors trying to copy files from /usr/local/share/bedrock-viz/data/

### ISSUE THREAD

**Issue by counterfitninja** [bedrock-viz/bedrock-viz#151]
Take a Clone of the latest High changes . Build and make solution. Inital run without the --XML files tried to load it from the /usr/local/share/bedrock-viz/ folder. 

If i pass in a calid --XML location the maps build but errors with the following. 

[  error] copyFileWithStringReplacement failed to open src (/usr/local/share/bedrock-viz/static/bedrock_viz.html.template) errno=No such file or directory(2)
[  error] copyFile failed for src (/usr/local/share/bedrock-viz/static/bedrock_viz.js) to dest (/home/drbird/bedoutput/bedrock_viz.js) error-code (2)
[  error] copyFile failed for src (/usr/local/share/bedrock-viz/static/bedrock_viz.css) to dest (/home/drbird/bedoutput/bedrock_viz.css) error-code (2)
[  error] copyDirToDir(src=/usr/local/share/bedrock-viz/static/js dest=/home/drbird/bedoutput/js) failed to open source directory
[   info] Copying icon images
[  error] copyDirToDir(src=/usr/local/share/bedrock-viz/static/images dest=/home/drbird/bedoutput/images) failed to open source directory

I would expect there files to either look at the local folder from there the Bedrock_vix was run from, or copy the files when compiling the solution. 
**Environment Information:**
 - OS: Windows Subsystem for Linux
 - bedrock-viz version: dynamic-top-and-bottom ( but i think its all Branches when using a new system)

**Comment by cabbey** [comment]
yeah, this is annoying as all get out. My current workaround for when I'm dealing with branches is that I have two symlinks in `/usr/local/share/bedrock-viz` that point to the `data` and `static` dirs in my git checkout. It's a hack, but it works.

The problem is it really only impacts those of us hacking on bedrock-viz, no users are impacted by it... and there's been plenty of actual user impacting TLC needed....

**Comment by cabbey** [comment]
In an ideal world, the binary would do a search for all of these files, looking along a list of paths until it found one. First it would look in the current working directory, then in the bin dir, and then in the system wide installed path. Annoyingly Windows *does* use the one from the bin dir... but that seems to be the only place it looks.
